mysite/_config/theme.yml

```diff
@@ -0,0 +1,7 @@
+---
+Name: cwptheme
+---
+SilverStripe\View\SSViewer:
+  themes:
+    - 'starter'
```

Comment:
Also we can add `$public` new pseudo-theme. Goes above `starter`.

Reply:
Can do. But you also please elaborate (or point to the new docs for 4.1) as to what & why?

Reply:
Required detail is linked here: https://github.com/silverstripe/cwp-installer/issues/6, specifically https://docs.silverstripe.org/en/4/changelogs/4.1.0/#use-new-public-theme-set

Reply:
Thanks :)